### PR TITLE
Fix game filters not applying

### PR DESCRIPTION
**Pull request by ACrazyTown**
FlxGame filters need to be set via the `setFilters()` function otherwise they'll get [overwritten in update](https://github.com/HaxeFlixel/flixel/blob/ed5300078914bf12705bd04e51b052d399bbefaa/flixel/FlxGame.hx#L772)

### CODE REVIEW

**Comment by Geokureli** [comment]
So previously we were setting it directly, [here](https://github.com/HaxeFlixel/flixel-demos/blob/dev/Effects/Filters/source/PlayState.hx#L113). If this is bad practice we should take it out and maybe find a way to throw a warning in flixel

**Comment by Geokureli** [comment]
I created https://github.com/HaxeFlixel/flixel/pull/3293

File: Effects/Filters/source/PlayState.hx [hunk 1]
```diff
@@ -131,6 +131,8 @@ class PlayState extends FlxState
 		checkbox.callback = function()
 		{
 			FlxG.camera.filtersEnabled = !checkbox.checked;
+
+			FlxG.game.setFilters(filters);
```
Comment (Geokureli) [inline Effects/Filters/source/PlayState.hx]: I say revert this change and replace line 113 with a setFilters call

**Review by Geokureli — CHANGES_REQUESTED** [review]

File: Effects/Filters/source/PlayState.hx (hunk 1 above)
Comment (ACrazyTown) [reply Effects/Filters/source/PlayState.hx]: done

**Review by Geokureli — APPROVED** [review]

**Comment by Geokureli** [comment]
Thanks!
